src/js/base/ripe.js

```diff
@@ -668,6 +669,7 @@ ripe.Ripe.prototype.setOptions = function(options = {}) {
     this.usePrice = this.options.usePrice === undefined ? !this.noPrice : this.options.usePrice;
     this.noDiag = this.options.noDiag === undefined ? false : this.options.noDiag;
     this.useDiag = this.options.useDiag === undefined ? !this.noDiag : this.options.useDiag;
+    this.size = this.options.size || null;

```

Comment:
Default to `1000`: https://github.com/ripe-tech/ripe-sdk#options

Reply:
I have this defaulting to null in other places
https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/visual/image.js#L53
https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/visual/configurator-prc.js#L51

should I also default to 1000 instead?

Reply:
No need I believe, they'll just inherit from here.